VERSION

```diff
@@ -1 +1 @@
-0.4.0-dev
```

Comment:
Why are there two commits that both touch `VERSION`, with the second cancelling out the change the first one made to the file?